contracts/FlowToken.cdc

```diff
@@ -96,6 +117,59 @@ pub contract FlowToken: FungibleToken {
         return <-create Vault(balance: 0.0)
     }
 
+    pub fun getViews(): [Type] {
+        return [Type<FungibleTokenMetadataViews.FTView>(),
+                Type<FungibleTokenMetadataViews.FTDisplay>(),
+                Type<FungibleTokenMetadataViews.FTVaultData>()]
+    }
+
+    /// Get a Metadata View from FlowToken
+    ///
+    /// @param view: The Type of the desired view.
+    /// @return A structure representing the requested view.
+    ///
+    pub fun resolveView(_ view: Type): AnyStruct? {
+        switch view {
+            case Type<FungibleTokenMetadataViews.FTView>():
+                return FungibleTokenMetadataViews.FTView(
```

Comment:
Maybe some view to `totalSupply ` can also be added

Reply:
I'll make a PR to add that to `FungibleTokenMetadataViews` then add that here

Reply:
https://github.com/onflow/flow-ft/pull/139